packages/components/inputs/select-utils/src/create-select-styles.ts

```diff
@@ -58,6 +64,17 @@ type TState = {
   isSelected?: boolean;
 };
 
+type TDesignTokenName = keyof typeof designTokens;
+
+const getHorrizontalConstraintValue = (value?: THorizontalConstraint) => {
+  if (!value) return null;
+  const tokenName = `constraint${value}`;
+  if (tokenName in designTokens) {
+    return designTokens[tokenName as TDesignTokenName];
+  }
+  return null;
+};
```

Comment:
If we return `null`, then we will have that as the value of a CSS property, which is invalid.

I would suggest to keep this simple and return a default value in case a null param is provided (which should not happen as it's checked in this function consumers).

```suggestion
const getHorizontalConstraintValue = (value?: THorizontalConstraint) => {
  return designTokens[`constraint${value}` as TDesignTokenName] || designTokens.constraint3;
};
```